lib/js/src/compose/types/chart/util.ts

```diff
@@ -241,6 +255,67 @@ export const hasRelativeDisplay = ({ type }: KV) => isRadialChart({ type })
 // Makes a standarised alias from modifier or dimension report option
 export const makeAlias = ({ alias, aggregate, modifier, field }: Metric) => alias || `${aggregate || modifier || 'none'}_${field}`.toLocaleLowerCase()
 
+export function formatChartValue (value: string | number, formatConfig?: FormatData): string {
+  let n: number | string = 0 || ''
+  // if value contains alphabetic chars parseFloat() will return NaN
+  // and n will equal 0
+  const containsAlphabeticChars = isNaN(Number(value))
+  let result = ''
+
+  if (!containsAlphabeticChars) {
```

Comment:
Instead of redefining the function here, i suggest moving the number one to some utils, and using that one here aswell. So we don't redefine it multiple times